packages/postcss/src/types.ts

```diff
@@ -9,6 +9,7 @@ export interface UnoPostcssPluginOptions {
     apply: string
     screen: string
     theme: string
+    unocss: string
```

Comment:
What is unocss directive?

Reply:
The placeholder: `@unocss` ,  `@unocss preflights` 

https://github.com/unocss/unocss/tree/main/packages/postcss#unocss

Reply:
> allows customizing unocss at-rule and plugin name. this is useful if you want to have separate CSS files and uno configs for different routes, for example, marketing site and dashboard.

For example:

```css
/* marketing.css */
@marketing;
````

```css
/* dashboard.css */
@dashboard;
````

I may remove this feature later, I'm not super sure about this. That's why I kept it undocumented. It still lacks VS Code support. I will later explore how we could support multiple configs in VS Code.